application/Profile/LocalProfile.py

```
@@ -88,33 +94,41 @@ def add_session(self, host, username, conn=None):
                                             public_key_comment=this_private_key_path)
             if not status:
+                logger.warning("Failed to save RSA SSH private key in %s", this_private_key_path)
                 return status, reason
 
         self.save_profile()
 
+        logger.info('Successfully saved RSA SSH private key')
         return True, ''
 
-    def delete_session(self, session_id):
+    def delete_session(self, session_id) -> tuple[bool, str]:
```

Ruff (0.8.2)

Missing type annotation for function argument `session_id`

(ANN001)